assets/lang/sklmessages_pl_PL.properties

```diff
@@ -30,39 +30,39 @@ game.status.launching=Uruchamianie...
 game.status.playing=W grze...
 game.status.idle=Idle
 
-login.username=Mojangowy Adres Email
+login.username=Mojangowy adres e-mail
 login.username.offline=Nick
 login.password=Has\u0142o
 login.offline=Offline
 login.login=Zaloguj
-login.existing.name=Zalogowani U\u017cytkownicy
+login.existing.name=Zalogowani u\u017cytkownicy
 login.existing.play=Graj
 login.existing.logout=Wyloguj
 
 editor.cancel=Anuluj
 editor.delete=Usu\u0144
 editor.browse=Otw\u00f3rz folder gry
 editor.save=Zapisz Profil
-editor.visibility=Widoczno\u015b\u0107 Launchera
+editor.visibility=Widoczno\u015b\u0107 launchera
 editor.resolution=Rozdzielczo\u015b\u0107
-editor.gamedir=Folder Gry
+editor.gamedir=Folder gry
 editor.beta=Beta
-editor.alpha=Alfa
+editor.alpha=Alpha
 editor.snap=Snapshoty
-editor.version=U\u017cyj wersji
+editor.version=U\u017cyj wersji:
 editor.latest=U\u017cywaj najnowszej wersji
-editor.exe=Executable
-editor.jvm=JVM Arguments
-editor.copy=Kopia
+editor.exe=Plik wykonywalny:
+editor.jvm=Argumenty JVM:
+editor.copy=Kopia: 
```

Comment:
Profile names with `:` looks a little bit weird

Reply:
After a colon, you can use words in the nominative case. If wasn't included, then you would get things like "Kopia mój profil" etc.